app/Helpers/CreateVersionPolygonGeometryHelper.php

```diff
@@ -53,7 +48,7 @@ public static function createVersionPolygonGeometry(string $uuid, $geometry)
 
             return response()->json(['message' => 'Failed to create site polygon version.'], 500);
         } catch (\Exception $e) {
-            return response()->json(['error' => 'An error occurred: ' . $e->getMessage()], 500);
+            throw $e;
```

Comment:
Was this for testing? If you're intentionally letting the exception bubble up to the HTTP layers, then there's no reason for the try / catch.

Reply:
try catch removed